src/seedu/addressbook/parser/Parser.java

```diff
@@ -54,7 +54,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
you need to update where `parseCommand` was previously called as an instance member.